tests/unit/amo/pages/TestCollection.js

```diff
@@ -1392,6 +1371,67 @@ describe(__filename, () => {
       }),
     );
   });
+  it('sends a server redirect when username is not lower case', () => {
```

Comment:
Should this description be about the case not matching, as opposed to "not lower case"?

Reply:
ah right